### ISSUE TITLE
PSU 3.7.8 - Dashboards not starting on boot

### ISSUE THREAD

**Issue by MattHarrisUltima** [ironmansoftware/powershell-universal#2008]
### Steps to Reproduce

1. Move from Docker image ```ironmansoftware/universal:3.7.7-ubuntu-20.04``` to ```ironmansoftware/universal:3.7.8-ubuntu-20.04```

Reverting to Docker image ```ironmansoftware/universal:3.7.7-ubuntu-20.04``` allows the container to start as expected.

### Expected behavior

```console
Dashboards start
```

### Actual behavior

```console
1st Dashboard on list hangs on 'Starting'
```

### Environment data

Docker container: ironmansoftware/universal:3.7.8-ubuntu-20.04

### Visuals

# Behaviour on 3.7.7
![2023-01-27 09_58_26-PSU - Admin Console](https://user-images.githubusercontent.com/106333485/215063524-dc0cd37c-197c-41b5-a696-af9546c7a212.png)

# Behaviour on 3.7.8
![2023-01-27 10_10_33-PSU - Admin Console](https://user-images.githubusercontent.com/106333485/215063536-d99030ae-c53c-47fa-834c-1b3d6612b2b1.png)

# Debug Logs
[log20230127.txt](https://github.com/ironmansoftware/issues/files/10517856/log20230127.txt)

**Comment by adamdriscoll** [comment]
Do you have any errors in the dashboard log? 

<img width="420" alt="image" src="https://user-images.githubusercontent.com/1256531/215100830-d5c5928e-6da9-457e-ace4-244fd3b2a258.png">

**Comment by MattHarrisUltima** [comment]
Hi @adamdriscoll,

I am re-deploying now to take a look. I will update you shortly.

**Comment by MattHarrisUltima** [comment]
Hi @adamdriscoll,

The logs are empty, I'm afraid:
![2023-01-27 13_49_07-PSU - Admin Console](https://user-images.githubusercontent.com/106333485/215102404-d5d0f3de-b1fd-4920-8f49-bae993e58ef7.png)
![2023-01-27 13_48_18-PSU - Admin Console](https://user-images.githubusercontent.com/106333485/215102406-8be37300-aac2-4e2f-a4e8-dacef1e15b6b.png)

**Comment by MattHarrisUltima** [comment]
I have performed some further investigations and removed all files from the dashboard folder, replacing them with a file with a Basic uddashboard and card pulled from another instance. Strangely enough, I still get the same error. Whenever I attempt to restart the dashboard, I receive a HTTP 500 alert inside the GUI.

Has anything else changed in 3.7.8 (possibly with kestrel) which relies on any type of config file within the root folder?

**Comment by adamdriscoll** [comment]
We didn't make any changes to the web server in 3.7.8. I'm assuming you are using persistently mounted storage volume for your dashboard? 

I tried just pulling the latest docker container and I can create and start dashboards in it.

**Comment by MattHarrisUltima** [comment]
Hi Adam,

I think I've figured out what's going on here.

After querying my SQL DB I found the following entries:
![image](https://user-images.githubusercontent.com/106333485/215742541-669ec588-2216-4e7d-889c-e799e84fd86a.png)


Looking at the environments page, I found this:
![image](https://user-images.githubusercontent.com/106333485/215742694-7c641b80-ce85-42f0-846e-d6ec624a9a50.png)
![image](https://user-images.githubusercontent.com/106333485/215742722-cd7c0ad5-046a-44c4-b355-3466d21ac195.png)


It appears when the switch from 7.3.1 to 7.3.2 occurred, the dashboards could not load, which explains why the dashboards were loading when I reverted.

Does the integrated environment use version of PowerShell at /user/bin/pwsh by when it loads up Universal.Server?
